api/interservice/authz/v2/project.proto

```diff
@@ -76,6 +74,7 @@ service Projects {
     rpc ListRules(ListRulesReq) returns (ListRulesResp) {};
     rpc ListRulesForProject(ListRulesForProjectReq) returns (ListRulesForProjectResp) {};
     rpc DeleteRule(DeleteRuleReq) returns (DeleteRuleResp) {};
+    rpc ListProjectRules(ListProjectRulesReq) returns (ProjectCollectionRulesResp) {};
```

Comment:
Not your change (I see it's from the old proto) but would be good to have the req/resp in alignment

Reply:
yeah totally i'm gonna rename the whole API 👍

Reply:
any movement on the value of "ListRulesForProject" and whether it could just move into the "GetProject" call? (If it's indeed only used on the detail project view of the UI).

Reply:
not yet, maybe we can chat at standup?